tests/JsonParseErrorExceptionTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/*
+    Copyright 2014 Rustici Software
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+use TinCan\JsonParseErrorException;
+
+class JsonParseErrorExceptionTest extends PHPUnit_Framework_TestCase
+{
+
+    private $exception;
+
+    private $malformedValue  = '.....';
+    private $jsonErrorNumber;
+    private $jsonErrorMessage;
+
+    public function setUp()
+    {
+        $this->jsonErrorNumber = JSON_ERROR_SYNTAX;
+        $this->jsonErrorMessage = 'Syntax error, malformed JSON';
+        $this->exception = new JsonParseErrorException(
+            $this->malformedValue,
+            $this->jsonErrorNumber,
+            $this->jsonErrorMessage
+        );
+    }
+
+    public function testThatWeCanGetOurInformationBackOut()
+    {
+        $this->assertEquals($this->malformedValue, $this->exception->malformedValue());
+        $this->assertEquals($this->jsonErrorNumber, $this->exception->jsonErrorNumber());
+        $this->assertEquals($this->jsonErrorMessage, $this->exception->jsonErrorMessage());
+    }
+
+    public function testThatWeGetAMeaningfulErrorMessage()
```

Comment:
That doesn't seem descriptive enough as "Get" has lots of definitions (around 40-50). Although I do think that my method name here is a little too descriptive. testThatErrorsAreMeaningful()?